lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,185 @@
+
+#### Audience Callback
+
+As noted above in the `config` object created at script setup, there is a callback method named `onProfileReady` that is called after targeting completes. This work happens asynchronously, so the callback is fired when the targeting work is completed.
+
+When fired, the callback is passed a `profile` object that has the following functionality.
+
+Method Name | Return Type | Parameters | Description
+----------- | ----------- | ---------- | -----------
+`getProfileId()` | String | n/a | The id of the profile associated with the browser
+`getAudienceString( [delimiter=','], [limit])` | String | 1. `delimiter` - optional, string, defaults to comma. <br/> 2. `limit` - optional, integer, defaults to no limit | Delimited string of targeting codes or audience ids, based on your established tag settings. The default, i.e. no arguments, is comma delimited with no limit. Example: `"aud1,aud2"` or `"1,2"`
+`getAudiences([limit])` | Array | `limit` - optional, integer, defaults to no limit | Array of either targeting codes as strings or audience ids as integers, based on your established tag settings. Example: `["aud1","aud2"]` or `[123,789]`
+
+#### Local Storage Usage
+
+As noted above in the `config` object created at script setup, there is a parameter named `audienceLocalStorage`. If passed, when targeting is completed, the audiences are written out to the browser's localStorage.
+
+Value | Result
+:---: | ------
+`false` | This is the default, and audiences are not written to browser localStorage.
+`true` |  Lightning Tag writes audiences to localStorage under the key `lotame_CLIENT_ID_auds` (where CLIENT_ID is your Lotame clientId)
+`'string'` | Passing a string tells Lightning Tag to write the audiences to localStorage under the key of the string you pass. (i.e. `audienceLocalStorage: 'myLotameAudiences'`)
+
+Audiences are written out as a string of comma-delimited targeting codes or audience IDs, based on your established tag settings. Example: `"aud1,aud2"` or `"123,789"`
+
+#### Tag Ready Callback
+
+The `onTagReady` callback fires when the Lotame Lightning Tag interface is ready to be used. Which includes below described methods such as `collect` and `page`.
+
+```javascript
+onTagReady: function(namespace) {
+  // Lightning Tag script is ready to be used
+},
+```
+
+?> This event does not confirm audiences are available, that is the `onProfileReady` callback described above in this document.
+
+#### Auto Run Flag
+
+The `autoRun` boolean is defaulted `true` and runs the Lotame data collection and targeting retrieval is run immediately after the script is loaded. By default it is `true` so those functions run automatically after the script is loaded. If set to `false` control is left to the client's website to fire data collection and targeting using the `collect()` and `page()` methods described lower on this document.
```

Comment:
```The `autoRun` boolean is defaulted `true` and runs the Lotame data collection and targeting retrieval immediately after the script is loaded. If set to `false` control is left to the client's website to fire data collection and targeting using the `collect()` and `page()` methods described later on this document.```

changed some wording, there were artifacts of prior versions

Reply:
Much better.